src/sentry/web/frontend/debug/mail.py

```diff
@@ -371,6 +371,123 @@ def digest(request):
 
 
 @login_required
+def report(request):
```

Comment:
nitpick: it'd be nice to avoid adding more code to this giant file (e.g. move the view to its own file like all the other newer code does)